GoogleUtilitiesComponents.podspec

```diff
@@ -0,0 +1,39 @@
+Pod::Spec.new do |s|
+  s.name             = 'GoogleUtilitiesComponents'
+  s.version          = '1.0.0'
+  s.summary          = 'Google Utilities Component Container for Apple platforms.'
+
+  s.description      = <<-DESC
+An internal Google utility that is a dependency injection system for libraries to depend on other
+libraries in a type safe and potentially weak manner.
+Not intended for direct public usage.
+                       DESC
+
+  s.homepage         = 'https://github.com/firebase/firebase-ios-sdk/tree/master/GoogleUtilitiesComponents'
```

Comment:
Needs to be reachable to pass pod lib lint

Reply:
Changed, thanks.